databases/week1/design.sql

```diff
@@ -0,0 +1,10 @@
+SELECT COUNT(*) AS task_count FROM tasks;
+SELECT COUNT(*) AS task_count FROM tasks WHERE due_date IS NULL ;
+SELECT * FROM tasks WHERE status = 'done';
```

Comment:
With this one, I am wondering if the task table in your database  has 'status' column? It seems that the task table in the lesson data doesn't contain 'status' column - it has 'status_id' though, which is integer and not a string. 

If your table doesn't contain 'status' column, the query won't work. If it is the case. I would suggest making it to the following:

SELECT * FROM tasks WHERE status_id = 3;

Reply:
Hi Alina, thank you so much for your feedback. That was my fault that you confused about my data, I didn't save my queries in the computer and just write it in a vscode, But now I fix it my errors and wrote again and all of them are work. Now I try again to put it in my vscode and send it in github.